vmm/src/config.rs

```diff
@@ -320,6 +320,8 @@ pub struct CpusConfig {
     pub topology: Option<CpuTopology>,
     #[serde(default)]
     pub kvm_hyperv: bool,
+    #[serde(default)]
+    pub phys_bits: Option<u8>,
```

Comment:
This is the user's *desired* physical bits, or `max_phys_bits` because it's the maximum number of bits that the caller is ready to use with its workload. I think the logic should be that the actual physical bits that we're going to use is:

If `max_phys_bits` is `None` then `actual_phys_bits = get_host_cpu_phys_bit()` else `actual_physical_bits = min(config.max_phys_bits, get_host_cpu_phys_bits())`.

Then `actual_phys_bits` would be the number that you'd use for the memory allocator and the CPUID patch.

Reply:
Done!